website/docs/docs/core/pip-install.md

```diff
@@ -154,3 +154,72 @@ python -m pip install \
 ```
 
 Or, better yet, just install the package(s) you need!
+
+<VersionBlock firstVersion="1.8">
+
+### Install prereleases of dbt adapters
+
+To install prerelease versions of dbt Core and your adapter, use this command:
+
+```shell
+python3 -m pip install --pre dbt-core dbt-adapter-name
+````
+We recommend you install prereleases in a [virtual Python environment](https://packaging.python.org/en/latest/guides/installing-using-pip-and-virtual-environments/). You can run the commands to activate and install your virtual environment. You’ll need to activate your virtual environment before you start installing or using packages within it. For example, to install a prerelease in a `POSIX bash`/`zsh` virtual Python environment, use the following command:
+
+```shell
+dbt --version
+python3 -m venv venv
+source venv/bin/activate
+python3 -m pip install --upgrade pip
+python3 -m pip install --pre dbt-core dbt-adapter-name
```

Comment:
This is the key part, and it looks good 👍 

```shell
python3 -m pip install --pre dbt-core dbt-adapter-name
```

Note: it will also install any pre-releases of **all** dependencies.